daisy_workflows/image_import/ubuntu/translate.py

```diff
@@ -163,6 +165,39 @@ def DistroSpecific(g):
   g.command(['update-grub2'])
 
 
+def add_gce_repositories(g, release):
+  g.write(
+    '/etc/apt/sources.list.d/partner.list',
+    partner_repo.format(release=release))
+  if release in ['xenial', 'bionic']:
+    # The OS config agent is currently supported for Ubuntu 16.04 and 18.04.
+    # For adding 20.04, see b/161470431.
+    utils.update_apt(g)
+    utils.install_apt_packages(g, 'gnupg', 'wget')
+    g.command(
+      ['wget', 'https://packages.cloud.google.com/apt/doc/apt-key.gpg',
+       '-O', '/tmp/gce_key'])
+    g.command(['apt-key', 'add', '/tmp/gce_key'])
+    g.rm('/tmp/gce_key')
+    # OS Config agent is only supported for xenial and bionic.
+    # Installing via instructions:
+    #    https://cloud.google.com/compute/docs/manage-os
+    g.write('/etc/apt/sources.list.d/google-compute-engine.list',
+            os_config_repo.format(release=release))
+
+
+def install_gce_packages(g, release):
+  logging.info('Installing GCE packages.')
+  utils.update_apt(g)
+  pkgs = ['gce-compute-image-packages', 'google-cloud-sdk']
+  if release in ['xenial', 'bionic']:
+    # OS Config agent is only supported for xenial and bionic.
```

Comment:
Maybe have this as info log, same as for Windows?

Reply:
good idea! Updated here as well as in Debian (which doesn't support debian 8).